src/ui/server/controllers/insurance/business/company-details/index.ts

```diff
@@ -204,8 +204,13 @@ const post = async (req: Request, res: Response) => {
       });
     }
 
+    const updateBody = {
+      ...body,
+      ...company,
+    };
+
     // if no errors, then runs save api call to db
-    const saveResponse = await mapAndSave.companyDetails(req.body, application);
+    const saveResponse = await mapAndSave.companyDetails(updateBody, application);
```

Comment:
Do  we have an e2e test to spy on the save function? Seems test would fail with this change